slides/src/content/component.md

```diff
@@ -61,14 +61,14 @@ export class ToDoListComponent implements OnInit {
 
 ## Including a Component
 
-- Refer to component by putting its selector tag in something else's HTML
-  - Remove the completed list and summary to make things fit on the slide
+- Refer to the component by putting its `selector` in another component's template
```

Comment:
No back quotes around the word "selector" in this context (just as you wouldn't back-quote words like "tag" or "attribute").